samplesheet/fqtk-samplesheet.csv

```diff
@@ -0,0 +1,2 @@
+id,samplesheet,fastq
+test,nf-core/test-datasets/demultiplex/testdata/sim-data/fqtk_sample_metadata_subset.tsv,nf-core/test-datasets/raw/demultiplex/testdata/sim-data/fqtk_subset_fastqs.tar.gz
```

Comment:
Should these be the raw GitHub links?

Reply:
I had copied the format used in sgdemux-samplesheet.csv, if it is better to have them as GitHub links I am happy to change that!

Reply:
I think they have to be 🤔 I guess I missed that in sgdemux-samplesheet.csv. I'm not sure how that's working in the tests then... Check the `b2fq-samplesheet.csv` as an example.

Reply:
I have changed sgdemux and fqtk to reflect the format in `b2fq-samplesheet.csv`. As for the tests, I cannot find a situation where the sample sheets are being taken directly from GitHub so that possibly explains it...

Reply:
Yeah we left the nf-core samplesheets in the demultiplex repo until they settle, since they might get updated often and we're lazy. The "tool" samplesheet (looks like `[read_structure_manifest.csv](https://github.com/nf-core/test-datasets/pull/758/files#diff-006cb1611a27159b17ef261a05607d041e212f28b800cc1d6cac7d4496e58199)` in this case) can live here definitely as that probably won't change.